src/ui/modules/VideoHero.tsx

```
@@ -0,0 +1,311 @@
+  onBackClick,
+}: {
+  error: string | null;
+  type?: string;
+  onBackClick: () => void;
+}) => {
+  return (
+    <div className="flex items-center justify-center h-full bg-black text-white text-center p-4">
+      <div>
+        <p className="text-xl font-semibold mb-2">Video Error</p>
+        <p>{error || `Could not find a valid video ID for this ${type || ''} video.`}</p>
+        <button onClick={onBackClick} className="mt-4 px-4 py-2 bg-white text-black rounded">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)